pyvocz/views.py

```diff
@@ -163,7 +176,6 @@ def event(series_slug, date_slug):
     query = query.options(joinedload(tables.Event.talks))
     query = query.options(joinedload(tables.Event.venue))
     query = query.options(joinedload(tables.Event.talks, 'talk_speakers'))
-    query = query.options(joinedload(tables.Event.links))
```

Comment:
It's an optimization: the data is loaded in the initial SQL query, rather than when the attribute is first accessed.